Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -81,6 +81,7 @@ void VideoConfig::Load(const std::string& ini_file)
   settings->Get("DumpPath", &sDumpPath, "");
   settings->Get("BitrateKbps", &iBitrateKbps, 2500);
   settings->Get("InternalResolutionFrameDumps", &bInternalResolutionFrameDumps, false);
+  settings->Get("EnableGPUTextureDecoding", &bEnableGPUTextureDecoding, 0);
```

Comment:
false/true instead of 0.
And I'm for enabling it by default. AFAIK the only reason to make it optional is the lack of testing, so enable it by default, and you'll get your testing ;) Else it will just be delayed...